README.md

```diff
@@ -7,14 +7,36 @@ WASM runtime.
 
 ### Dependencies
 
-- Go 1.22.6
-- Tinygo v0.32.0
+- Go (see `go.mod` for version)
+- Tinygo v0.34.0
+- jq
 - golangci-lint
 - [easyjson](https://github.com/mailru/easyjson) v0.9.0
     - Used for generating JSON serialization code
+- `bky-as` - Blocky Attestation Service
+    - The SDK is designed to work with the Blocky Attestation Service
+    - The version compatible with this SDK is pinned in the `shell.nix` file.
+
+Additional project dependencies are specified in tbe `shell.nix` file.
 
 ### Development
 
+### Nix Shell
+To enter a development shell with all dependencies, run:
+
+```bash
+nix-shell --pure
```

Comment:
`--pure` seemed like the appropriate choice here. Any objections?

Reply:
I think it is fine.

WRT your question in the PR description:
> the linter only works in --pure mode for me, something to do with how C is getting accessed, likely some interaction between my local environment and nix. Does make lint work for you when not in pure mode (nix-shell)?

For me, it works even when I am not in pure mode, but I am not sure if that is too telling.  Because I am using nixos, and I have very little in my environment, the difference between `nix-shell` and `nix-shell --pure` is pretty negligible.

Reply:
ok, good to know. I will try to continue tracking this as I work with nix and see if I can figure it out, but using `--pure` seems like an ok solution for now since we are likely going to use it in --pure mode in CI and whenver we are using nix to run a command and for any dev who wants the makefile commands to "just work"